### PR TITLE
Customizable directory for rules files

### PR DESCRIPTION
**Pull request by eRadical**
We need support for custom directory for rules file.

### CODE REVIEW

**Comment by paulfantom** [comment]
This changes where are those files stored on deployer host not on target, so why not just symlink it in a proper directory. File search path is not absolute so a file doesn't need to be stored inside a role directory.

**Comment by hamishforbes** [comment]
Hi,

Can we re-open this? I have the same issue.
The role is installed via `ansible-galaxy` and therefore not checked in to my git repo.

I have my alerting rules as static files outside of the roles directory but no way to tell this role to load them from there as the fileglob is hard coded.

My workaround is to have an extra `copy` task in my playbook, after importing the `cloudalchemy.prometheus` role.
But this also means I have to manually trigger a reload on the prometheus service too, as I can't access the handler inside the role.

The cloudalchemy Grafana role solves this in the same way as this PR:
https://github.com/cloudalchemy/ansible-grafana/blob/master/defaults/main.yml#L194

IMO this is a much nicer approach than having to get my colleagues to run ansible-galaxy and then manually symlink stuff into the role directory, or having duplicate tasks/reloads.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,42 @@
 # Change Log
```
Comment (paulfantom) [inline CHANGELOG.md]: This file is created automatically and every manual change to it is overwritten. Could you remove it from PR?

File: README.md [hunk 2]
```diff
@@ -24,12 +28,13 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 
 | Name           | Default Value | Description                        |
 | -------------- | ------------- | -----------------------------------|
-| `prometheus_version` | 2.5.0 | Prometheus package version. Also accepts `latest` as parameter. Only prometheus 2.x is supported |
+| `prometheus_version` | 2.7.1 | Prometheus package version. Also accepts `latest` as parameter. Only prometheus 2.x is supported |
```
Comment (paulfantom) [inline README.md]: This is not related to a PR.

**Comment by paulfantom** [comment]
I think your rebase went badly and you included commits which shouldn't be included in this PR. Could you fix it?

**Comment by eRadical** [comment]
Yes... it went badly.
I recreated the branch... if you can reopen the request it should be fine now.

**Comment by paulfantom** [comment]
I cannot since a branch was removed :(
